hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/LifecycleITest.groovy

```diff
@@ -105,18 +106,16 @@ class LifecycleITest extends AbstractITestBase {
         assertEquals(200, resp.status)
 
         // Send in avail data to fire the trigger
-        // Note, the groovyx rest client seems incapable of allowing a JSON payload and a TEXT response (which is what
-        // we get back from the activemq rest client used by the bus), so use the bus' java rest client to do this.
-        RestClient busClient = new RestClient(host, port);
-        String json = "{\"data\":[{\"id\":\"test-autodisable-avail\",\"timestamp\":" + System.currentTimeMillis() +
-                      ",\"value\":\"DOWN\",\"type\":\"availability\"}]}";
-        busClient.postTopicMessage("HawkularAlertData", json, null);
-        //assertEquals(200, resp.status)
+        String jsonData = "{\"availability\":" +
+                "[{\"id\":\"test-autodisable-avail\",\"timestamp\":" + System.currentTimeMillis() +
+                      ",\"value\":\"DOWN\",\"type\":\"AVAILABILITY\"}]}";
+        resp = client.post(path: "data", body: jsonData);
+        assertEquals(200, resp.status)
 
         // The alert processing happens async, so give it a little time before failing...
         for ( int i=0; i < 10; ++i ) {
-            // println "SLEEP!" ;
-            Thread.sleep(500);
+            println "SLEEP!" ;
+            Thread.sleep(1000);
```

Comment:
When I executed the mvn clean install -Prest or -Pstandalone profile I noticed that some times (1 time in 9 in a row) tests are failing due some bad result.
I tend to think that in some cases there is some lag somewhere (probably due embedded C*).
Increasing the sleep helps to avoid this issue.